[[ Bug 22004 ]] Fix memory leak when configuring syntax highlighting

runrevmark opened the pull request:
This patch fixes a memory leak in the internal 'ide script configure'
command, caused by a failure to release a string used to create a name
value. The issue has been fixed by using the AndRelease form of
MCNameCreate.

livecodeali commented on engine/src/ide.cpp:
```diff
@@ -535,7 +535,7 @@ void MCIdeScriptConfigure::exec_ctxt(MCExecContext &ctxt)
 
                 MCStringFormat(t_attr_key_string, "%s attributes", s_script_keywords[t_keyword]);
-                MCNameCreate(t_attr_key_string, &t_attr_key);
+                MCNameCreateAndRelease(t_attr_key_string, &t_attr_key);
```
While you're here, these should probably be marked `/* UNCHECKED */`

runrevmark replied:
Done!

runrevmark committed:
[[ Bug 22004 ]] Fix memory leak when configuring syntax highlighting

This patch fixes a memory leak in the internal 'ide script configure'
command, caused by a failure to release a string used to create a name
value. The issue has been fixed by using the AndRelease form of
MCNameCreate.

livecodeali commented:
@livecode-vulcan review ok 61831d9

livecode-vulcan commented:
💙 review by @livecodeali ok 61831d91635906fbeb98616d1a0d242942179928

livecode-vulcan commented:
😎 test success 61831d91635906fbeb98616d1a0d242942179928

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/447)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/439)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/439)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/439)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/439)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1961)
* try-community-universal-ios-iphoneos12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.1/builds/189)
* try-community-universal-ios-iphonesimulator12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.1/builds/188)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1992)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1982)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1981)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2009)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1950)
